keyboards/phdesign/phac/keyboard.json

```
@@ -0,0 +1,57 @@
+{
+    "manufacturer": "PHDesign",
+    "keyboard_name": "phac",
+    "maintainer": "nonameCCC",
+    "bootloader": "rp2040",
+    "bootmagic": {
+        "matrix": [0,5]
+    },
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true,
+        "encoder": true
+    },
+    "matrix_pins": {
+        "direct": [
+            ["GP6","GP5","GP4","GP3","GP2","GP1","GP0"]
+        ]
+    },
+    "processor": "RP2040",
+    "url": "",
+    "usb": {
+        "device_version": "2.2.0",
+        "pid": "0x0001",
+        "vid": "0x5048"
+    },
+    "layouts": {
+        "LAYOUT": {
+            "layout": [
+                {"matrix": [0, 0], "x": 0, "y": 1},
+                {"matrix": [0, 1], "x": 1.5, "y": 1},
+                {"matrix": [0, 2], "x": 3, "y": 1},
+                {"matrix": [0, 3], "x": 4.5, "y": 1},
+                {"matrix": [0, 4], "x": 0.375, "y": 2},
+                {"matrix": [0, 5], "x": 2.5, "y": 0},
+                {"matrix": [0, 6], "x": 3.375, "y": 2}
+            ]
```

The order and key sizes here does not match the physical layout, and should be updated so that it follows the left to right, top to bottom convention.

```suggestion
            "layout": [
                {"matrix": [0, 0], "x": 3.25, "y": 0},
                {"matrix": [0, 1], "x": 0, "y": 1, "w":1.5},
                {"matrix": [0, 2], "x": 2, "y": 1, "w":1.5},
                {"matrix": [0, 3], "x": 4, "y": 1, "w":1.5},
                {"matrix": [0, 4], "x": 6, "y": 1, "w":1.5},
                {"matrix": [0, 5], "x": 0.5, "y": 2, "w":3},
                {"matrix": [0, 6], "x": 4, "y": 2, "w":3}
            ]
```

The sizes here are mostly a guess from your provided image.

---

Any reason for why you chose not to apply the key width changes? They are not all 1u in the product image.

---

Sorry about forgotten to add width information with those keys. will fix that pretty soon.

---

Now proper key width was set in the kayboard.json file.